README.md

```diff
@@ -0,0 +1,11 @@
+# The Final Project for Northeaster University Accelerated Data Analytics Professional Certification Program
+
```

Comment:
Add a paragraph text that explains what Northeastern is and what criteria the project had. You can copy and paste from Canvas or write it in your own words